Server/src/main/webapp/resources/js/controller.js

```diff
@@ -0,0 +1,172 @@
+let model;
```

Comment:
Fix code style. I'll not describe what should you fix from code style side. We discussed it several times